### PR TITLE
Add uptime check

### PR DESCRIPTION
**Pull request by ajmaddaford**
Adds the uptime check that is currently manually created in production. 

Configuring SSL certificate validation is not currently possible in Terraform - https://github.com/terraform-providers/terraform-provider-google-beta/pull/1243.

The conditions config is derived from https://cloud.google.com/monitoring/alerts/policies-in-json.

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -35,3 +35,7 @@ variable "project_team" {
   description = "The team this project belongs to, for billing and reporting purposes, etc"
   default     = "eq"
 }
+
+variable "uptime_check_host" {
```
Comment (MebinAbraham) [inline variables.tf:39]: Should add this to `terraform.tfvars.example` since it is mandatory.

File: monitoring_alerts.tf [hunk 2]
```diff
@@ -10,6 +10,31 @@ resource "google_monitoring_notification_channel" "slack_alert" {
   }
 }
 
+resource "google_monitoring_uptime_check_config" "http" {
+  display_name = "${var.uptime_check_host}"
+  timeout      = "5s"
+  period       = "60s"
+
```
Comment (MebinAbraham) [inline monitoring_alerts.tf:18]: I had to add `project      = "${var.stackdriver_workspace}"` for this to work.

File: monitoring_alerts.tf [hunk 3]
```diff
@@ -10,6 +10,32 @@ resource "google_monitoring_notification_channel" "slack_alert" {
...
+    path    = "/status"
+    port    = 443
+    use_ssl = true
+  }
+
+  monitored_resource {
+    type = "uptime_url"
+
+    labels = {
+      project_id = "${var.stackdriver_workspace}"
```
Comment (MebinAbraham) [inline monitoring_alerts.tf]: This isn't needed and I think we should remove it.

**Review by MebinAbraham — DISMISSED** [review]

File: monitoring_alerts.tf (hunk 2 above)
Comment (ajmaddaford) [reply monitoring_alerts.tf:18]: Fixed. Mine worked as my project id is the same as my stackdriver workspace.

**Review by JamesGardiner — CHANGES_REQUESTED** [review]
Needs an uptime alert policy

**Review by MebinAbraham — COMMENTED** [review]
Can you `terraform fmt` the files.

**Review by MebinAbraham — APPROVED** [review]

**Review by JamesGardiner — APPROVED** [review]

File: monitoring_alerts.tf (hunk 3 above)
Comment (ajmaddaford) [reply monitoring_alerts.tf]: Ok, the Terraform doc isn't clear and it's in their example. Removed.

**Pull request merged**